app/views/api/work_orders/_latest_appointment.html.erb

```diff
@@ -19,7 +19,7 @@
     <tr>
       <th scope="row">Supplier:</th>
       <td>
-        <%= work_order.repair_request.work_orders.supplier_reference %>
+        <%= work_order.repair_request.work_orders.first.supplier_reference %>
```

Comment:
Was this broken before?

Reply:
Well, `work_orders` was a single `Hackney::Supplier`. I changed it to be an array of `WorkOrder`, so the `.first` is needed now.

Reply:
@spannerinworks